README.md

```diff
@@ -93,6 +93,27 @@ try {
 }
 ```
 
+## Basic usage of `exads-api-client-php` client with API token authentication
+
+
+```php
+<?php
+
+require_once 'vendor/autoload.php';
+
+try {
+    $client = new Exads\Client('https://api.exads.com/url/');
+    $apiToken = $client->login->getToken('APItoken');
```

Comment:
how about simply adding this line in the example above or below `$apiToken = $client->login->getToken('username', 'password');` with a comment saying that there are 2 ways to get the auth token?